xml/hwbp_registers.xml

```diff
@@ -148,24 +180,7 @@
             will never fire (unless consecutive instructions match the
             appropriate triggers).
         </field>
-        <field name="action" bits="17:12" access="R/W" reset="0">
-            Determines what happens when this trigger matches.
-
-            0: Raise a breakpoint exception. (Used when software wants to use
-            the trigger module without an external debugger attached.)
-
-            1: Enter Debug Mode. (Only supported when \Fdmode is 1.)
-
-            2: Start tracing.
-
-            3: Stop tracing.
-
-            4: Emit trace data for this match. If it is a data access match,
-            emit appropriate Load/Store Address/Data. If it is an instruction
-            execution, emit its PC.
-
-            Other values are reserved for future use.
-        </field>
+        <field name="action" bits="17:12" access="R/W" reset="0" />
```

Comment:
I think here you say something like "this is action, as defined by Table X"

Reply:
Done.